WORKSPACE.bazel

```diff
@@ -949,3 +949,26 @@ http_file(
 load("//rs/tests:kubeconfig.bzl", "kubeconfig")
 
 kubeconfig()
+
+http_archive(
+    name = "bitcoin_core",
```

Comment:
we need to reconcile this and what we have in the [third_party directory](https://github.com/dfinity/ic/blob/master/third_party/bitcoin-core/BUILD.bitcoin-core.bazel).

Is it possible to remove altogether what we have in `master/third_party` ?

Reply:
[Dropped]([f833361](https://github.com/dfinity/ic/pull/1491/commits/f833361fa472c1ac8752d02528101cc39cafd059)) and
```
bazel test //rs/bitcoin/adapter:adapter_integration
```
passes locally. No other test should be affected so let's see if CI indeed passes.

Reply:
do note that this means we won't build it from src AFAICT; not saying this is a problem, just FYI

Reply:
cc: @tthebst @stedfn since they added it if I remember correctly

Reply:
@mraszyk made moot by https://github.com/dfinity/ic/pull/1603 correct?